rto_node/src/OmniDriveROS.cpp

```diff
@@ -70,4 +77,14 @@ void OmniDriveROS::setMaxMin(double max_linear_vel, double min_linear_vel, doubl
 	min_linear_vel_ = min_linear_vel;
 	max_angular_vel_ = max_angular_vel;
 	min_angular_vel_ = min_angular_vel;
-}
\ No newline at end of file
+}
+
+bool OmniDriveROS::handleSetOmniDriveEnabled(const std::shared_ptr<rto_msgs::srv::SetOmniDriveEnabled::Request> request,
+                                    std::shared_ptr<rto_msgs::srv::SetOmniDriveEnabled::Response> response)
+{
+    enabled_ = request->enabled;
```

Comment:
enabled_ = request->enable, change the request variable name.

Reply:
oh true, thank you!